sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from sympy.core.singleton import S
+Reals = S.Reals
+del S
```

Comment:
It's a good idea to `del` names that you don't want to export in `__init__.py` files.